app/models/core_data_connector/media_content.rb

```diff
@@ -2,6 +2,7 @@ module CoreDataConnector
   class MediaContent < ApplicationRecord
     # Includes
     include Identifiable
+    include Manifestable
```

Comment:
Does `Manifestable` come from one of the gems we're using?

Reply:
No, it's a [concern](https://github.com/performant-software/core-data-connector/blob/e680acee1b00e36a5b1bc14d5f51733e552114c5/app/models/concerns/core_data_connector/manifestable.rb) in this repo that defines a polymorphic relationship between the parent record and one or more manifests.